java-checks/src/main/resources/org/sonar/l10n/java/rules/java/S1860_java.html

```diff
@@ -1,13 +1,23 @@
 <p>Objects which are pooled and potentially reused should not be used for synchronization. If they are, it can cause unrelated threads to deadlock
 with unhelpful stacktraces. Specifically, <code>String</code> literals, and boxed primitives such as Integers should not be used as lock objects
-because they are pooled and reused. The story is even worse for <code>Boolean</code> objects, because there are only two instances of
```

Comment:
Why removing the part about Boolean, it was making a good example, and does not prevent to have the rest